### ISSUE TITLE
Nextflow fails on Altair's PBSPro

### ISSUE THREAD

**Issue by cerebis** [cerebis/meta-sweeper#8]
Related to #9 

Nextflow uses a directive to qsub which does not exist in the implementation by Altair. I suspect they've only tested the codebase on Adaptive Computings PBS/Torque variant.

Eg. Nextflow .command.run preamble for qsub contains `-d` which is not supported in PBSPro.

```
#PBS -d /a/work/dir/
#PBS -N nf-Evolve_(3)
#PBS -o /a/work/dir/.command.log
#PBS -j oe
#PBS -V
```

Instead, processes will need to use `cd $PBS_O_WORKDIR` in a "beforeScript" directive. Additionally, I will need to be able to disable the unsupported qsub `-d` option.

**Comment by koadman** [comment]
bummer. 
can we work within the GVL/CloudMan/slurm environment for the time being?

**Comment by cerebis** [comment]
It took some unwrapping of the onion so to speak, but I have pin-pointed what the problem is with Nextflow and PBSPro.

First, they are using the workdir directive, which PBSpro doesn't support. This is actually a hugely annoying short coming of PBSPro and I cannot believe they don't have a switch for this and rely on the user CD'ing with the environment variable $PBS_O_WORKDIR. Really dumb...

Anyway, even when you address this issue, jobs still will not start. This turns out to be related to the choice of script names. I will make a new issue.

**Comment by cerebis** [comment]
Support for this has been implemented on our fork. Closes both #4 and #9
